### PR TITLE
Run pipeline script

### PR DESCRIPTION
**Pull request by IsaackMwenda**

### CODE REVIEW

File: Pipfile [hunk 1]
```diff
@@ -5,8 +5,8 @@ name = "pypi"
 
 [packages]
 CoreDataModules = {editable = true,ref = "v0.9.1",git = "https://www.github.com/AfricasVoices/CoreDataModules"}
-PipelineInfrastructure = {editable = true,ref = "08cfb07be4c80656924bceaeeaef15a21ddb7a72", git = "https://www.github.com/AfricasVoices/PipelineInfrastructure"}
-RapidProTools = {editable = true,git = "https://www.github.com/AfricasVoices/RapidProTools",ref = "dfaa9c2284671e65a08662a77eb819ca5b1d7fa7"}
+PipelineInfrastructure = {editable = true,ref = "v0.0.1", git = "https://www.github.com/AfricasVoices/PipelineInfrastructure"}
```
Comment (as2388) [inline Pipfile:8]: As of this morning you can use v0.0.2 :) (better logging)

File: docker-run.sh [hunk 2]
```diff
@@ -100,31 +100,31 @@ fi
 docker start -a -i "$container"
 
 # Copy the output data back out of the container
-echo "Copying out traced data..."
+echo "copying traced data from '$container:/data/output.json' to '$OUTPUT_JSON'"
```
Comment (as2388) [inline docker-run.sh:103]: Are you logging for copies into the container too?

File: docker-run.sh [hunk 3]
```diff
@@ -100,31 +100,31 @@ fi
...
 mkdir -p "$(dirname "$OUTPUT_JSON")"
 docker cp "$container:/data/output.json" "$OUTPUT_JSON"
 
-echo "Copying out ICR..."
+echo "copying ICR files from '$container:/data/output-icr/' to '$OUTPUT_ICR_DIR'"
```
Comment (as2388) [inline docker-run.sh]:
```suggestion
echo "Copying ICR files from '$container:/data/output-icr/' to '$OUTPUT_ICR_DIR'"
```

**Review by as2388 — APPROVED** [review]

File: Pipfile (hunk 1 above)
Comment (IsaackMwenda) [reply Pipfile:8]: Nice thanks

File: docker-run.sh (hunk 2 above)
Comment (IsaackMwenda) [reply docker-run.sh:103]: I should probably do that too thanks

**Pull request merged**